bin/citrea/tests/e2e/mod.rs

```diff
@@ -1090,21 +1090,6 @@ async fn test_soft_confirmations_status_one_l1() -> Result<(), anyhow::Error> {
     // submitted by sequencer.
     wait_for_l1_block(&da_service, 3, None).await;
 
-    // now retrieve confirmation status from the sequencer and full node and check if they are the same
-    for i in 1..=6 {
-        let status_node = full_node_test_client
-            .ledger_get_soft_confirmation_status(i)
-            .await
-            .unwrap();
-
-        assert_eq!(SoftConfirmationStatus::Trusted, status_node.unwrap());
```

Comment:
For the record, the sync on full node was happening faster in a way which gets those blocks to a `Finalized` state rather than `Trusted`. Therefore, i opted for removing these checks as we're going to check for finalized state at the end of this test.

Reply:
we can check for trusted state before a sequencer commitment is sent maybe

Reply:
Restored